tests/lib/rules/key-spacing.js

```diff
@@ -150,6 +150,17 @@ ruleTester.run("key-spacing", rule, {
             afterColon: true
         }]
     }, {
+        code: [
+            "obj = { key ",
+            "    :     ",
+            " longName };"
+        ].join("\n"),
+        options: [{
+            beforeColon: true,
+            afterColon: false,
+            mode: "minimum"
```

Comment:
When `align` isn't set, the rule never warns on properties that span multiple lines - see `isSingleLine`. Putting the property key and value on the same line should validate the code to handle `minimum`.